cluster-up/ssh.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -z "$KUBEVIRTCI_PATH" ]; then
+    KUBEVIRTCI_PATH="$(
+        cd "$(dirname "$BASH_SOURCE[0]")/"
+        echo "$(pwd)/"
+    )"
+fi
+
+source ${KUBEVIRTCI_PATH}/hack/common.sh
+
+test -t 1 && USE_TTY="-it"
+source ${KUBEVIRTCI_PATH}/cluster/$KUBEVIRT_PROVIDER/provider.sh
+source ${KUBEVIRTCI_PATH}/hack/config.sh
+
+ssh_key=${KUBEVIRTCI_PATH}/hack/common.key
+chmod 600 $ssh_key
+
+node=$1
+
+if [ -z "$node" ]; then
+    echo "node name required as argument"
+    echo "okd example: ./ssh master-0"
+    echo "k8s example: ./ssh node01"
+    exit 1
+fi
+
+if [[ $provider_prefix =~ okd.* ]]; then
+    ports=$($KUBEVIRTCI_PATH/cluster/cli.sh --prefix $provider_prefix ports --container-name cluster)
+
+    if [[ $node =~ worker-0.* ]]; then
+        port=$(echo "$ports" | grep 2202 | awk -F':' '{print $2}')
+    elif [[ $node =~ master-0.* ]]; then
+        port=$(echo "$ports" | grep 2201 | awk -F':' '{print $2}')
+    fi
+
+    if [ -z "$port" ]; then
+        echo "no ssh port found for $node"
+        exit 1
+    fi
+    ssh -lcore -p $port core@127.0.0.1 -i ${ssh_key}
+else
+    ${_cli} --prefix $provider_prefix ssh "$1"
```

Comment:
We should unify the access between okd and the other clusters. I understand that we are not that far yet.

Reply:
agreed, that would be nice.